src/javascripts/components/mushroomList/mushroomList.js

```diff
@@ -50,17 +53,23 @@ const showShroomForm = (e) => {
 const addShroomEvent = (e) => {
   e.preventDefault();
 
+  const file = document.getElementById('mush-image').files[0];
+  const image = file.name;
```

Comment:
get the image from the input. `file` is the image itself, and `image` is just the name of the image